src/structures/APIMessage.js

```diff
@@ -74,6 +74,16 @@ class APIMessage {
     return this.target instanceof Message;
   }
 
+  /**
+   * Whether or not the target is an interaction
+   * @type {boolean}
+   * @readonly
+   */
+  get isInteraction() {
+    const Interaction = require('./Interaction');
```

Comment:
You should import it in the top level scope or make it an expandable Structure ?

Reply:
Pretty sure having the require in the function is for lazy loading purposes. Various other functions in APIMessage do this